src/test/java/com/flipkart/zjsonpatch/JsonDiffTest.java

```diff
@@ -127,44 +127,33 @@ public void testPath() throws Exception {
         Assert.assertEquals(target, expected);
     }
 
-
```

Comment:
Tests have updated. Removed the `asJsonNullSafe` method. Since the existing `asJson` is enriched with the new contract, it's tests have too.